---
identifier: "CVE-2018-6188"
identifiers:
- "GHSA-rf4j-j272-fj86"
- "CVE-2018-6188"
package_slug: "pypi/django"
title: "Exposure of Sensitive Information to an Unauthorized Actor"
description: "django.contrib.auth.forms.AuthenticationForm in Django , allows remote
  attackers to obtain potentially sensitive information by leveraging data exposure
  from the confirm_login_allowed() method, as demonstrated by discovering whether
  a user account is inactive."
date: "2021-09-17"
pubdate: "2018-10-03"
affected_range: ">=2.0.0,<2.0.2||>=1.11.8,<1.11.10"
fixed_versions:
- "2.0.2"
- "2.0.2"
affected_versions: "All versions starting from 2.0.0 before 2.0.2, all versions starting
  from 1.11.8 before 1.11.10"
not_impacted: "All versions before 2.0.0, all versions starting from 2.0.2, all versions
  before 1.11.8, all versions starting from 1.11.10"
solution: "Upgrade to versions 2.0.2, 2.0.2 or above."
urls:
- "https://nvd.nist.gov/vuln/detail/CVE-2018-6188"
- "https://github.com/advisories/GHSA-rf4j-j272-fj86"
- "https://usn.ubuntu.com/3559-1/"
- "https://www.djangoproject.com/weblog/2018/feb/01/security-releases/"
- "http://www.securitytracker.com/id/1040422"
cvss_v2: "AV:N/AC:L/Au:N/C:P/I:N/A:N"
cvss_v3: "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N"
uuid: "3e8094f7-ed60-4825-943f-0158b6ec64c6"
cwe_ids:
- "CWE-1035"
- "CWE-200"
- "CWE-937"
